### PR TITLE
Use new ChartStore cache for line chart api; Update to use human reada…

### PR DESCRIPTION
**Pull request by shifucun**
…ble stats var everywhere.

This removes mmethod, scaling factor and unit from the cache key.

### CODE REVIEW

**Comment by shifucun** [comment]
@googlebot I signed it!

File: server/models/datachart_handler.py [hunk 1]
```diff
@@ -302,14 +299,16 @@ def get_ancestor_places(dcid):
...
+  keys = [dcid + '^count^measured^^^^Person' for dcid in dcids]
   chart_data = datacommons.get_chart_data(keys)
   for key, data in chart_data.items():
     dcid = key.split('^')[0]
-    for date, v in data['obsTimeSeries']['val'].items():
+    for date, v in data['data'].items():
       result[(dcid, date)] = v
   return result
 
+import logging
```
Comment (beets) [inline server/models/datachart_handler.py:310]: remove?

File: server/models/datachart_handler.py [hunk 2]
```diff
@@ -338,16 +336,17 @@ def get_plot_data(place_args, pc, gr):
   chart_data = datacommons.get_chart_data(list(keys))
   result = []
   for key, data in chart_data.items():
-    dcid = key.split('^')[0]
-    points = [(date, v) for date, v in data['obsTimeSeries']['val'].items()]
+    logging.info(data)
```
Comment (beets) [inline server/models/datachart_handler.py:339]: remove

File: static/js/stats_var.test.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+import { STATS_VAR_TEXT } from "./stats_var";
+import USChartConfig from "./chart_config.json";
+
+
+test("stats var names", () => {
+  for (let section of USChartConfig) {
+    for (let chart of section.charts) {
+      for (let statsVar of chart.statsVars) {
+        expect(STATS_VAR_TEXT[statsVar]).toBeTruthy();
```
Comment (beets) [inline static/js/stats_var.test.ts:26]: nice!

Comment (shifucun) [reply static/js/stats_var.test.ts:26]: :)

**Pull request merged**